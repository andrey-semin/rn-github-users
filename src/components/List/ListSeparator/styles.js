import { StyleSheet } from 'react-native'

import { colors } from 'styles'

export default StyleSheet.create({
  separator: {
    height: 1,
    flex: 1,
    backgroundColor: colors.grey
  }
})
