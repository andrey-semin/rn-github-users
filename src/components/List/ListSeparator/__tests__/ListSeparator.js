import React from 'react'
import renderer from 'react-test-renderer'

import ListSeparator from '../ListSeparator'

it('ListSeparator renders correctly', () => {
  const tree = renderer.create(<ListSeparator />).toJSON()
  expect(tree).toMatchSnapshot()
})
