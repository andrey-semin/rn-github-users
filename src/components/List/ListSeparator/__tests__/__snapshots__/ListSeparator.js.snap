// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListSeparator renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#f2f2f2",
      "flex": 1,
      "height": 1,
    }
  }
/>
`;
